### PR TITLE
Retrieve replies to specified post (Medhya goel)

### PR DESCRIPTION
**Pull request by medhyaGoel**
Retrieves replies (oldest first) to a specified post. Can return all or a specified number of replies and can specify whether or not to include only direct replies to the original post.

### CODE REVIEW

File: truthbrush/api.py [hunk 1]
```diff
@@ -140,8 +143,10 @@ def _get_paginated(self, url: str, params: dict = None, resume: str = None) -> A
             # Will also sleep
             self._check_ratelimit(resp)
 
-    def userLikes(self, post: str, top_num: int = 40) -> bool | Any:
-        """Return the top_num most recent users who liked the post."""
+    def userLikes(
+        self, post: str, includeAll: bool = False, top_num: int = 40
```
Comment (KonradIT) [inline truthbrush/api.py:147]: Maybe better to name this variable `include_all` to keep in line with the snake case naming convention throughout the code.

File: truthbrush/api.py [hunk 2]
```diff
@@ -154,9 +159,36 @@ def userLikes(self, post: str, top_num: int = 40) -> bool | Any:
...
                 n_output += 1
-                if n_output >= top_num:
+                if not includeAll and n_output >= top_num:
                     return
 
+    def pull_comments(
+        self,
+        post: str,
+        includeAll: bool = False,
+        onlyFirst: bool = False,
```
Comment (KonradIT) [inline truthbrush/api.py:169]: Same as before. 

Maybe better to name these variables `include_all` and `only_first` to keep in line with the snake case naming convention throughout the code.

File: truthbrush/cli.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 import click
 from datetime import date
 import datetime
-
+import pdb
```
Comment (KonradIT) [inline truthbrush/cli.py:8]: Why was this imported?

**Comment by medhyaGoel** [comment]
Hi @KonradIT, I updated the casing and deleted the unnecessary imports.

File: truthbrush/api.py [hunk 4]
```diff
@@ -140,8 +143,10 @@ def _get_paginated(self, url: str, params: dict = None, resume: str = None) -> A
             # Will also sleep
             self._check_ratelimit(resp)
 
-    def userLikes(self, post: str, top_num: int = 40) -> bool | Any:
-        """Return the top_num most recent users who liked the post."""
+    def userLikes(
+        self, post: str, include_all: bool = False, top_num: int = 40
+    ) -> bool | Any:
+        """Return the top_num most recent (or all) users who liked the post."""
```
Comment (KonradIT) [inline truthbrush/api.py:149]: perhaps for a different PR, but could userLikes be changed to user_likes?

**Comment by medhyaGoel** [comment]
@KonradIT snake_cased userLikes(), as well

**Pull request merged**